docs/modules/operation/pages/quick-start/thresholding.adoc

```diff
@@ -0,0 +1,181 @@
+* Absolute change - changes by the specified amount (for example, on a fiber-optic link, a change in loss of anything greater than 3 dB is a problem regardless of what the original or final value is).
+* Relative change - changes by percent (for example, available disk space changes more than 5% from the last poll).
+
+These thresholds can be basic (tested against a single value) or an expression (evaluated against multiple values in an expression).
+This Quick Start guide shows how to create both kinds of threshold.
+
+[[threshold-basic]]
+== Create a basic threshold
+
+This procedure describes how to create a basic threshold for a single, system-wide variable: the number of logged-in users.
+This threshold tells {page-component-title} to create an event when the number of logged-in users on the device exceeds two, and re-arm when it falls below two.
+
+Before creating this threshold, make sure you have at least two users defined in your {page-component-title} instance (see <<quick-start/users.adoc#create-user, Create a new user>>).
+
+. Click the *Gear* symbol in the top-right of the screen.
+. Under *Performance Measurement*, choose *Configure Thresholds*.
++
+A screen with a list of preconfigured threshold groups appears.
+We will work with `netsnmp`.
+For information on how to create a threshold group, see xref:deep-dive/thresholds/thresh-group.adoc[Create a Threshold Group] in the Deep Dive section.
+. Click *Edit* beside the `netsnmp` group.
+. Click *Create New Threshold* at the bottom of the *Basic Thresholds* area of the screen.
+. Set the following information and click *Save*:
+
+[options="header"]
+[cols="1,3,2"]
+|===
+| Field
+| Description
+| Input value
+
+| Type
+| Triggers an event when the data source value equals or exceeds the threshold value, and re-arms when it equals or drops below the re-arm value.
+| high
+
+| Datasource
+| Name of the data source you want to threshold against.
+For this tutorial, we have provided the data source for logged-in users.
+For information on how to determine a metric's data source, see xref:operation:deep-dive/thresholds/datasource.adoc#datasource-determine[Determine the data source] in the Deep Dive section.
+| hrSystemNumUsers
+
+
+| Datasource label
+| Optional text label.
+Not required for this Quick Start guide.
+| Leave blank.
+
+| Value
+| The value above which you want to trigger an event.
+In this case, we want to trigger an event when the number of logged-in users exceeds two.
+| 2
+
+| Re-arm
+| The value below which we want the system to re-arm.
+In this case, once the number of logged-in users falls below two.
+| 2
+
+| Trigger| The number of consecutive times the threshold value can occur before the system triggers an event.
+Since our default polling period is 5 minutes, a value of 3 means {page-component-title} would create a threshold event if there are more than 2 users for 15 minutes.
```

Comment:
```suggestion
Since our default polling period is five minutes, a value of three means {page-component-title} would create a threshold event if there are more than two users logged in for 15 minutes.
```